packages/wonder-blocks-button/components/button-core.js

```diff
@@ -10,9 +10,9 @@ import type {SharedProps} from "./button.js";
 
 type Props = SharedProps &
     ClickableHandlers & {
-        hovered: boolean,
-        focused: boolean,
-        pressed: boolean,
+        hovered?: boolean,
+        focused?: boolean,
+        pressed?: boolean,
```

Comment:
These aren't used in the `ButtonCore` examples in docs.md.

Reply:
Hmm - it seems like we should just change the examples in docs.md rather than changing the structure of this component (we should probably always be specifying these properties - or provide defaults via defaultProps).

Reply:
I'll update to use `defaultProps`.